build.gradle.kts

```diff
@@ -179,7 +179,9 @@ dependencies {
 
     testImplementation(libs.mockito.junit.jupiter)
     testRuntimeOnly(libs.junit.platform.launcher)
+    testCompileOnly(libs.checker.qual)
 
+    integrationTestImplementation(libs.mockito.junit.jupiter)
```

Comment:
We don't need this, do we?

```suggestion
```

Reply:
I overlooked. removed with other changes.